### PR TITLE
Adds @implements and @interface tags

### PR DESCRIPTION
**Pull request by mlzummo**
Adds tags to suppress warnings on doc build. Can be parsed out as well from json docs

Signed-off-by: Mathew Zummo <mzummo@gmail.com>

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by okuryu** [comment]
I'm not able to verify that this fixes the warning. Did you tested this?

**Comment by mlzummo** [comment]
yes i tested it, a complete implementation of @interface @implements would be nice but this will work for now, should I open a issue for that?

**Comment by mlzummo** [comment]
image

**Comment by mlzummo** [comment]
i see the prob, i made a good will update pR

File: lib/docparser.js [hunk 1]
```diff
@@ -170,7 +170,9 @@ YUI.add('docparser', function (Y) {
             'url',          // project url(s)
             'uses',         // 0..n compents mixed (usually, via augment) into the prototype
             'value',        // the value of a constant
-            'writeonce'     // YUI attribute config
+            'writeonce',    // YUI attribute config
+            'implements',  // Implements Interface
+            'interface'    // Is Interface
```
Comment (okuryu) [inline lib/docparser.js]: Can you make alphanumerics sort?

**Comment by mlzummo** [comment]
@okuryu should be good to go

**Comment by okuryu** [comment]
Merged. Thanks!

**Pull request merged**